bridge-token-factory/src/lib.rs

```diff
@@ -404,7 +440,6 @@ impl BridgeTokenFactory {
         Promise::new(bridge_token_account_id.parse().unwrap())
             .create_account()
             .transfer(BRIDGE_TOKEN_INIT_BALANCE)
-            .add_full_access_key(self.owner_pk.clone())
```

Comment:
How we use the full access key for owner before?

Reply:
This key was added to the token contract to be able to upgrade it and call private methods.

Reply:
This key was added to the token contract to be able to upgrade it and call private methods.
Now we are going to remove this PK access, so the token should be controlled just by the factory contract, and the factory contract will be controlled by DAO.